src/axis/axis.ts

```diff
@@ -1320,11 +1323,14 @@ export module LabelLayoutStrategy {
                 // TODO don't do these rotations if we already did them
                 axisLabel.style("text-anchor", "middle")
                     .attr(
-                    {
-                        "dx": "0em",
-                        "dy": "1em",
-                        "transform": "rotate(0)"
-                    });
+                        "dx", "0em"
+                    )
+                    .attr(
+                        "dy", "1em"
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line